test/ufixed.jl

```diff
@@ -122,7 +134,7 @@ x = 0xaauf8
 iob = IOBuffer()
 show(iob, x)
 str = takebuf_string(iob)
-@test startswith(str, "UFixed8(")
+@test startswith(str, "FixedPointNumbers.UFixed{UInt8,8}(")
```

Comment:
Can we turn this into `UFixed{UInt8,8}`? Since this gets used in Images.jl and the type names are already quite long.